geopy/geocoders/what3words.py

```diff
@@ -114,14 +107,11 @@ def geocode(
         """
 
         if not self._check_query(query):
-            raise exc.GeocoderQueryError(
-                "Search string must be 'word.word.word'"
-            )
+            raise exc.GeocoderQueryError("Search string must be 'word.word.word'")
 
         params = {
-            'addr': query,
-            'lang': lang.lower(),
```

Comment:
`Language` param seems to be missing now. Could you please return it?

Reply:
In v3, language param is no longer applicable for geocode queries. Please see here: https://developer.what3words.com/public-api/docs#convert-to-coords